src/negbinfit.jl

```diff
@@ -30,7 +30,13 @@ function mle_for_θ(y::AbstractVector, μ::AbstractVector, wts::AbstractVector;
         end
         θ = θ - δ
     end
-    converged || throw(ConvergenceException(maxIter))
+    if !converged
+        @warn """
```

Comment:
Triple quotes include line breaks. AFAIK you need to use normal quotes, possibly splitting message over several lines using `*`.

But I think we'd better pass the message to `ConvergenceException` than print a warning. Clearly that's not a warning, that's an error. And if the caller catches it, the warning shouldn't be printed to the output. Would you make a PR against StatsBase to add a `msg::String` field to `ConvergenceException`?